src/main/java/uk/gov/ons/census/fwmt/outcomeservice/service/impl/JobCacheManagerImpl.java

```diff
@@ -0,0 +1,23 @@
+package uk.gov.ons.census.fwmt.outcomeservice.service.impl;
+
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.ons.census.fwmt.outcomeservice.config.RedisUtil;
+import uk.gov.ons.census.fwmt.outcomeservice.service.JobCacheManager;
+
+
+@Slf4j
+@Service
+public class JobCacheManagerImpl implements JobCacheManager {
```

Comment:
I think stick this bit of functionality in CCSPLStore, and remove JobCacheManager & Imple

Reply:
This has been moved the CCSPLStore and the related job cache classes have been removed.